.github/workflows/pypi_build_and_images.yaml

```diff
@@ -0,0 +1,39 @@
+# This workflow will build and upload WMCore core services to the production PYPY
```

Comment:
Please update this description with what will actually happen. Build and upload to PyPi, then build and upload of a docker image to Gitlab registry.
In addition, PYPY should read PyPI.

Reply:
done